src/Core/hco_error_mod.F90

```diff
@@ -6,47 +6,74 @@
 ! !MODULE: hco_error_mod.F90
 !
 ! !DESCRIPTION: Module HCO\_Error\_Mod contains routines and variables
-! for error handling and logfile messages in HEMCO. It also contains
-! definitions of some globally used parameter, such as the single/double
-! precision as well as the HEMCO precision definitions. The HEMCO precision
-! is used for almost all HEMCO internal data arrays and can be changed below
-! if required.
-!\\
-!\\
-! The error settings are specified in the HEMCO configuration file and
-! error handling is performed according to these settings. They include:
-!
-! \begin{enumerate}
-! \item HEMCO logfile: all HEMCO information is written into the specified
-!     logfile. The logfile can be set to the wildcard character, in which
-!     case the standard output will be used (this may be another opened
-!     logfile).
-! \item Verbose: Number indicating the verbose level to be used.
-!     0 = no verbose, 3 = very verbose. The verbose level can be set in
-!     the HEMCO configuration file. The default value is 0.
-! \item Warnings: Number indicating the warning level to be shown.
-!     0 = no warnings, 3 = all warnings.
-! \end{enumerate}
-!
-! The error settings are set via subroutine HCO\_ERROR\_SET, called when
-! reading section 'settings' of the HEMCO configuration file (subroutine
-! Config\_ReadFile in hco\_config\_mod.F90). The currently active verbose
-! settings can be checked using subroutines HCO\_IsVerb and
-! HCO\_VERBOSE\_INQ. Messages can be written into the logfile using
-! subroutine HCO\_MSG. Note that the logfile actively need to be opened
-! (HCO\_LOGFILE\_OPEN) before writing to it.
-!\\
-!\\
-! The verbose and warning settings are all set to false if it's not the
-! root CPU.
-!\\
-!\\
-! As of HEMCO v2.0, all HEMCO error variables are organized in derived
-! type object HcoErr. HcoErr is a component of the HEMCO configuration
-! object (type ConfigObj, see hco\_types\_mod.F90). It must be passed
-! explicitly to all error routines. This design allows the invocation
-! of multiple independent HEMCO instances at the same time (which may
-! have different HEMCO error settings).
+! for error handling and logfile messages in HEMCO. It alse contains
+! definitions of some globally used parameters, such as single/double
```

Comment:
@lizziel: We should also update the ReadTheDocs accordingly in this section:
https://hemco.readthedocs.io/en/stable/hco-ref-guide/hemco-config.html#logfile

This can also be done as a separate commit to the `docs/dev` branch.

Reply:
Will do!